src/adapter/helpers.ts

```diff
@@ -41,7 +41,7 @@ export const findExistingJob = async (
       'The Smartling adapter requires a Smartling project identifier and a proxy URL. Please check your secrets document in this dataset, per the plugin documentation.',
     )
   }
-  const url = `https://api.smartling.com/jobs-api/v3/projects/${project}/jobs?jobName=${documentId}`
```

Comment:
The `jobName` QS filter has been removed because it breaks if the job does not use the `documentId` as job's name in Smartling. 

It fetches all jobs and filter by `referenceNumber` using the same logic.

Reply:
hey arthur! i would prefer this stay in the plugin for backwards compatibility. we've only added the additional logic to include the reference number a month or so ago, so other users may have old jobs that do not have a reference number encoded. ideally, this would be removed with a major version change, because it's technically a breaking change for users with in-progress jobs.

there is additional logic in the following few lines to use either the file URI or the reference number as a "catch." if you've noticed breaking behavior with this logic, let me know, but i'd rather catch both cases.